integreat_cms/cms/templates/poicategories/poicategory_list_row.html

```diff
@@ -25,6 +25,23 @@
             </div>
         </div>
     </td>
+    <td class="py-2 pr-2 text-left flex items-center gap-2">
+        {% if poi_category.icon %}
+            {# djlint:off H006 #}
+            <img src="{{ poi_category.icon }}"
+                 class="w-8 h-8 inline-block rounded-lg"
+                 alt="{{ poi_category.icon }}"/>
+            {# djlint:on #}
+            <div>{{ poi_category.get_icon_display }}</div>
+        {% endif %}
+    </td>
+    <td class="py-2 pr-2 text-left flex items-center gap-2">
+        {% if poi_category.color %}
+            <div class="w-8 h-8 inline-block rounded-lg"
+                 style="background-color: {{ poi_category.color }}"></div>
+            <div>{{ poi_category.get_color_display }}</div>
+        {% endif %}
+    </td>
```

Comment:
Wenn `flex` is used for both `<td>`, somehow they line up vertically under the tab `icon` 😞 the icon/color square and name won't stand side by side without `flex` though...